src/content/platform-overview/security/cluster-security/domain-allowlist/index.md

```diff
@@ -25,18 +25,26 @@ Below is a list of the external domains we require access to for our clusters to
 - amazonaws.com
     - domains:
         - `*.amazonaws.com`
+        - `ec2.eu-west-2.amazonaws.com`
+        - `sts.eu-central-1.amazonaws.com`
     - AWS services are used for a variety of tasks, such as etcd backup storage.
 - azurecr.io
     - domains:
         - `giantswarm.azurecr.io`
+        - `giantswarmpublic.azurecr.io`
```

Comment:
This registry contains our app catalogs. We could add it to the below description.